### DISCUSSION TITLE
Does Orchard core framework has a detailed documentation

### DISCUSSION THREAD

**Discussion by AbuDawood** [OrchardCMS/OrchardCore#10123] [Q&A]
I am going to begin developing a multi-tenant SaaS app using the OrchardCore framework. however, based on my investigation, the only available documentation is talking about how to develop a modular app and how to configure your project for multitenancy. are these all what the framework could provide ??

**Comment by ns8482e** [comment]
@AbuDawood  difficult to differentiate

However in  [docs/references](https://docs.orchardcore.net/en/latest/docs/reference/)  look for  - under "Core Modules" most of them are not related to CMS

**Reply by AbuDawood** [reply]
Thank you for your answer, It is understood now.
I have another question. 
in the provided sample for tenancy, they put the customerA and customerB configs in the appsetting file. I understand that modules are static and you can not make them installable in the admin dashboard??
Thanks in advance

**Reply by ns8482e** [reply]
Admin Dashboard is designed for CMS not for MVC
When you decide to use `ocmvc` (as per sample guide) you build everything your own.

However if you need to use admin dashboard and user management but don't want content management then use `occms` with `SaaS` recipe.
